collectors/info_collector.go

```diff
@@ -0,0 +1,60 @@
+package collectors
+
+import (
+	"context"
+
+	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// InfoCollector is a collector that keeps track of node information.
+type InfoCollector struct {
+	info *prometheus.Desc
```

Comment:
Is this actually useful in its current form? Given that the value is always zero, and AFAIK, there's no easy way to project only a label name into the graphs on Grafana.

Reply:
You can use these label values as single stats.